osc-domain/src/main/java/org/osc/core/broker/model/entities/virtualization/ServiceFunctionChain.java

```diff
@@ -36,34 +36,37 @@
 @Table(name = "SERVICE_FUNCTION_CHAIN", uniqueConstraints = { @UniqueConstraint(columnNames = { "name" }) })
 public class ServiceFunctionChain extends BaseEntity {
 
-	@Column(name = "name", nullable = false)
-	private String name;
+    @Column(name = "name", nullable = false)
+    private String name;
 
-	@ManyToMany(fetch = FetchType.LAZY)
-	@JoinTable(name = "SERVICE_FUNCTION_CHAIN_VIRTUAL_SYSTEM", joinColumns = @JoinColumn(name = "sfc_fk", referencedColumnName = "id"), inverseJoinColumns = @JoinColumn(name = "virtual_system_fk", referencedColumnName = "id"))
-	private Set<VirtualSystem> virtualSystems = new HashSet<>();
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(name = "SERVICE_FUNCTION_CHAIN_VIRTUAL_SYSTEM", joinColumns = @JoinColumn(name = "sfc_fk", referencedColumnName = "id"), inverseJoinColumns = @JoinColumn(name = "virtual_system_fk", referencedColumnName = "id"))
+    private Set<VirtualSystem> virtualSystems = new HashSet<>();
 
-	public ServiceFunctionChain(String name) {
-		this.name = name;
-	}
+    public ServiceFunctionChain(String name) {
```

Comment:
least to most complicated constructor.Move package private constructor below variable declarations